docs/package.json

```diff
@@ -9,9 +9,10 @@
     "postflutter:build": "yarn flutter:copy",
     "flutter:clean": "rm -rf public/flutter/authenticator",
     "flutter:copy": "cp -r ./flutter/authenticator/build/web public/flutter/authenticator",
-    "build": "yarn build:props-tables && next build",
+    "build": "yarn build:props-tables && yarn lint && next build",
```

Comment:
Should `lint` be in `build`? IMO it should be separate step, and we would explicitly call this in CI/CD or on Amplify Hosting.

Reply:
Good call. I'm thinking of changing it to a pre-commit instead in a future PR